display_list/display_list_image_filter.h

```diff
@@ -66,6 +68,12 @@ class DlImageFilter
     return From(sk_filter.get());
   }
 
+  std::shared_ptr<DlImageFilter> makeWithLocalMatrix(const SkMatrix&) const {
```

Comment:
I'm going to look into implementing this for real rather than just stubbing it out here. I think with this one addition to the DlImageFilter class hierarchy we can get rid of a few of the further uses of SkImageFilter down below.

I filed https://github.com/flutter/flutter/issues/108263 for this with a fairly high priority. I'll look to try to address it in the next couple of days.